### PR TITLE
Update and re-order blueprints by stars

### PR DESCRIPTION
**Pull request by mraible**
I'm tagging the stream leads for each blueprint in this update to make sure y'all still want to lead:

- Kotlin: @sendilkumarn ✅
- .NET Core: @nicolas63 ✅
- React Native: @ruddell
- Node.js: @amanganiello90 ✅
- Ionic: @mraible ✅
- Quarkus: @mraible ✅
- Micronaut: @atomfrede ✅
- Svelte: @vishal423 ✅
- JHipster Native: @mshima and @mraible ✅
- jOOQ: @mshima ✅
- Entity Audit: @mshima ✅

### CODE REVIEW

**Comment by amanganiello90** [comment]
Yes. I still manage the nodejs blueprint. Thanks

**Comment by mraible** [comment]
Unfortunately, I can't run this locally on my M1 to see if the table of contents `{:toc}` works. I was expecting a Netlify preview, but maybe that only happens when it's a PR from this repo?

**Comment by vishal423** [comment]
Yes, I will continue on Svelte blueprint

File: modules/official-blueprints.md [hunk 1]
```diff
@@ -12,42 +12,66 @@ sitemap:
...
+
+The Micronaut blueprint is supported and driven by people of the [Micronaut team](https://github.com/jhipster/generator-jhipster-micronaut/graphs/contributors) itself.
+It replaces all server side code with a [Micronaut](https://micronaut.io/) application.
+Source code and documentation can be found in the [generator-jhipster-micronaut](https://github.com/jhipster/generator-jhipster-micronaut) repository. {⭐️: 91}
+
+Stream lead is [Frederik Hahne](https://github.com/atomfrede).
+
+## Svelte
+
+The Svelte blueprint replaces the client with Svelete. Source code and documentation could be found in the [generator-jhipster-svelte](https://github.com/jhipster/generator-jhipster-svelte) repository. {⭐️: 44}
```
Comment (vishal423) [inline modules/official-blueprints.md:69]:
```suggestion
The Svelte blueprint replaces the client side code with SvelteKit framework. Source code and documentation could be found in the [generator-jhipster-svelte](https://github.com/jhipster/generator-jhipster-svelte) repository. {⭐️: 44}
```

**Comment by mraible** [comment]
@mshima Good idea. I added your suggestions.

File: modules/official-blueprints.md [hunk 2]
```diff
@@ -12,42 +12,78 @@ sitemap:
...
 
 * Enhance JHipster with new features using different languages and/or support
 * Demonstrate how the main generator behavior can be modified to fit anyone's needs
 
+The blueprints below are ordered by the number of stars they have on GitHub. 
+
 ## Kotlin
 
-It replaces most of Java back-end code with Kotlin. Source code and documentation could be found here: [repository](https://github.com/jhipster/jhipster-kotlin).
+The Kotlin blueprint replaces most of Java back-end code with Kotlin. Source code and documentation could be found in the [jhipster-kotlin](https://github.com/jhipster/jhipster-kotlin) repository. {⭐️: 411}
```
Comment (mraible) [inline modules/official-blueprints.md:24]:
```suggestion
The Kotlin blueprint replaces most of Java back-end code with Kotlin. Source code and documentation could be found in the [jhipster-kotlin](https://github.com/jhipster/jhipster-kotlin) repository. 
{⭐️: 411}
```

**Review by mraible — COMMENTED** [review]
Test to see if line break wraps line.

**Comment by mraible** [comment]
> There are 2 more blueprints https://github.com/jhipster/generator-jhipster-jooq and https://github.com/hipster-labs/generator-jhipster-entity-audit

@mshima Since you mention these, I'm assuming you're willing to be the stream lead and keep maintaining them? I added ✅ to indicate you are.

Still waiting for responses from @danielpetisme (.NET Core), @ruddell (React Native), @avdev4j (Quarkus), and @atomfrede (Micronaut). 

I'm particularly concerned about Quarkus since it hasn't had much activity. It says it's sponsored by Entando, yet it doesn't seem like they're doing much to support it. @avdev4j Should we adjust the description (to remove Entando) and find a new lead?

**Comment by danielpetisme** [comment]
Hi!

Regarding the .Net Core blueprint, the actual lead is @nicolas63. He's doing a great job managing the project's contributions.

When it comes to Quarkus... I personally gave up, I struggled too much following the updates of the blueprint mechanism./
I let @avdev4j speak for himself and Entando but on my side, the project should be deprecated and ultimately archived.

File: modules/official-blueprints.md [hunk 3]
```diff
@@ -12,42 +12,98 @@ sitemap:
...
-## Vue.js
+## .NET Core
 
-It replaces the whole front-end logic with Vue.js. Source code and documentation could be found here: [repository](https://github.com/jhipster/jhipster-vuejs).
+The .NET Core blueprint is the first attempt to leave the Java environment and join the .NET world. 
 
-Stream lead is: [Sahbi Ktifa](https://github.com/sahbi-ktifa).
+- Code and docs: [jhipster/jhipster-dotnetcore](https://github.com/jhipster/jhipster-dotnetcore)
+- GitHub ⭐️: 249
+- Stream lead: [Daniel Petisme](https://github.com/danielpetisme)
```
Comment (mraible) [inline modules/official-blueprints.md:36]:
```suggestion
- Stream lead: [Nicolas Raymond](https://github.com/nicolas63)
```

**Comment by mraible** [comment]
@danielpetisme I've updated this PR so @nicolas63 is the .NET Core lead. Nicolas, please confirm you're OK with this.

It's funny you should mention Quarkus. I have a slide in one of my presentations that mentions Micronaut and Quarkus blueprints. I haven't had to update it in a year because the number of releases, contributors, etc. hasn't changed. 

Micronaut's last release was [May 8, 2021](https://github.com/jhipster/generator-jhipster-micronaut/releases/tag/v1.0.2). Quarkus's last release was [August 23, 2021](https://github.com/jhipster/generator-jhipster-quarkus/releases/tag/v2.0.0-beta.1).  

Micronaut has been [upgraded](https://github.com/jhipster/generator-jhipster-micronaut/pull/265) for the new blueprint architecture. It also has an [open PR](https://github.com/jhipster/generator-jhipster-micronaut/pull/267) for upgrading to Micronaut 3. 

Quarkus has a lot of open issues, with several bug bounties assigned to them. It probably still needs to be upgraded to the latest blueprint architecture. I created an [issue](https://github.com/jhipster/generator-jhipster-quarkus/issues/245) and assigned a bug bounty to it.

I'm interested in seeing both Micronaut and Quarkus succeed as JHipster blueprints. I can take over as the Quarkus lead if no one else is interested.

**Comment by nicolas63** [comment]
i'm ok and i willl continue to support jhispter.net

**Comment by atomfrede** [comment]
Sorry @mraible for the delay. I will try to lead the micronaut part as good as I can. Need to try not to fall asleep with my daughter every night 😄

**Comment by mraible** [comment]
Thanks @atomfrede! We're down to two left: React Native and Quarkus.

**Comment by avdev4j** [comment]
hi @mraible 
I can't say I led the Quarkus because @danielpetisme really did it and I tried to help him in that way. As Daniel said we gave up a bit and that's true we don't really maintain the project anymore.

If we want this project to be maintained again, I suggest naming a new leader.

I will do a PR to update the Entando description.

**Comment by mraible** [comment]
@avdev4j Thanks for the update. I'll add myself as stream lead. If I'm not able to make anything happen by the end of 2022, we can ask the community for help, then archive the project if there are no volunteers.

File: modules/official-blueprints.md [hunk 4]
```diff
@@ -12,42 +12,98 @@ sitemap:
...
 
 ## Quarkus
 
-This blueprint supported, in part by the JHipster sponsor [Entando](https://www.entando.com/), and driven by the JHipster community for a [supersonic, subatomic Java back-end](https://quarkus.io/).
-Source code and documentation can be found here: [repository](https://github.com/jhipster/generator-jhipster-quarkus)
+The Quarkus blueprint is supported, in part by the JHipster sponsor [Entando](https://www.entando.com/), and driven by the JHipster community for a [supersonic, subatomic Java back-end](https://quarkus.io/).
+
+- Code and docs: [jhipster/generator-jhipster-quarkus](https://github.com/jhipster/generator-jhipster-quarkus)
+- GitHub ⭐️: 111
+- Stream lead: [Anthony Viard](https://github.com/avdev4j)
```
Comment (mraible) [inline modules/official-blueprints.md:68]:
```suggestion
- Stream lead: [Matt Raible](https://github.com/mraible)
```

**Pull request merged**